after/main.py

```diff
@@ -66,4 +63,4 @@ def main(cfg: MNISTConfig) -> None:
 
 
 if __name__ == "__main__":
-    main()
+    CLI(main)
```

Comment:
This is only [one of the ways](https://jsonargparse.readthedocs.io/en/stable/#basic-usage) in which jsonargparse can be used. It can also be used just like with argparse except that it has many more features. Complex parsers can be easily created from the signatures of functions and classes, see https://jsonargparse.readthedocs.io/en/stable/#classes-methods-and-functions.